cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -0,0 +1,617 @@
+---
+title: Provision a CockroachDB Cloud Cluster with Terraform
+summary: Learn how to provision a cluster using the CockroachDB Cloud Terraform provider.
+toc: true
+docs_area: manage
+---
+
+[Terraform](https://terraform.io) is an infrastructure-as-code provisioning tool that uses configuration files to define application and network resources. You can provision CockroachDB Cloud clusters and cluster resources by using the [CockroachDB Cloud Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach) in your Terraform configuration files.
+
+This tutorial shows you how to provision a CockroachDB Cloud cluster using the CockroachDB Cloud Terraform provider.
+
+<div class="filters clearfix">
```

Comment:
It's tough to tell from the source (or the preview) how Serverless and Dedicated are broken out. I assume there are two separate tabs.

Reply:
Yup, you can see that in the preview links in the Files Changed comment above.